docs/content/how-to-contribute/contribute-code.md

```diff
@@ -15,7 +15,7 @@ Apache Flink is maintained, improved, and extended by code contributions of volu
 ## Looking for what to contribute
 
 If you have a good idea for the contribution, you can proceed to [the code contribution process](#code-contribution-process).
-If you are looking for what you could contribute, you can browse open Jira issues in [Flink's bug tracker](https://issues.apache.org/jira/projects/FLINK/issues),
+If you are looking for what you could contribute, you can browse open Jira issues in [Flink's bug tracker](https://flink.apache.org/community.html#issue-tracker),
```

Comment:
There is a another link to jira further down in the text of this markdown. Additionally, there's also a Chinese version of this text which we could update.